### PR TITLE
Create Staff handbook page “leaving-dxw”

### PR DESCRIPTION
**Pull request by SamJCoop676**
Automatically generated by Netlify CMS

### CODE REVIEW

**Review by yndajas — COMMENTED** [review]
Not sure if this is still a draft, but a few quick suggestions (you might not be able to see the suggestions properly via email, but if you view it on GitHub you will)

File: src/staff-handbook/leaving-dxw.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+The People team will pick up the offboarding process from here, and guide you through it step by step. They will start by sending you a resignation acceptance letter that will:
+
+* say when your last working day will be, based on your contractual notice period
+* explain how much annual leave you have left 
+* let you know what your final payment date will be
+
+## What happens next 
+
+The People team will work with you on a plan to communicate that you’re leaving to the rest of dxw. But only if that’s what you want to do, they will be guided by you throughout the process. Departments that will need to be involved throughout your offboarding process, include Finance and Workplace Tech.
```
[the remark this answers is not in the document]
Comment (SamJCoop676) [reply src/staff-handbook/leaving-dxw.md:19]: yes Workplace Tech is what bob decided to label Matt's role as

[the remark this answers is not in the document]
Comment (SamJCoop676) [reply src/staff-handbook/leaving-dxw.md:19]: change to semi colon - But only if that’s what you want to do; they will be guided by you throughout the process.

happy to remove comma here - Departments that will need to be involved throughout your offboarding process include Finance and Workplace Tech.

[the remark this answers is not in the document]
Comment (yndajas) [reply src/staff-handbook/leaving-dxw.md:19]: Sounds good - I can't see this change yet. Is it still to be made?

[the remark this answers is not in the document]
Comment (SamJCoop676) [reply src/staff-handbook/leaving-dxw.md]: all updated now

[the remark this answers is not in the document]
Comment (yndajas) [reply src/staff-handbook/leaving-dxw.md]: Looks like the semicolon edit is there but I'm still seeing the comma in the last sentence

**Review by yndajas — APPROVED** [review]

**Pull request merged**